checks/signed_releases.go

```diff
@@ -66,16 +72,19 @@ func SignedReleases(c *checker.CheckRequest) checker.CheckResult {
 			}
 		}
 		if !signed {
-			c.Dlogger.Warn("release artifact %s not signed", r.TagName)
+			c.Dlogger.Warn3(&checker.LogMessage{
+				Text: fmt.Sprintf("release artifact %s not signed", r.TagName),
```

Comment:
Why not have path ?

Reply:
currently not supported by the repo client. I created https://github.com/ossf/scorecard/issues/890 for tracking.